Xamarin.Forms.Core/Stepper.cs

```diff
@@ -32,7 +32,11 @@ public class Stepper : View, IElementConfiguration<Stepper>
 		public static readonly BindableProperty ValueProperty = BindableProperty.Create("Value", typeof(double), typeof(Stepper), 0.0, BindingMode.TwoWay, coerceValue: (bindable, value) =>
 		{
 			var stepper = (Stepper)bindable;
-			return ((double)value).Clamp(stepper.Minimum, stepper.Maximum);
+			var result = ((double)value).Clamp(stepper.Minimum, stepper.Maximum);
+			var increment = stepper.Increment;
+			return Math.Abs(increment) > double.Epsilon
+				? Math.Round(result / increment) * increment
+				: result;
```

Comment:
Maybe we should actually do this the other way around to avoid the division if it has reached the bounds:

```csharp
if (value >= max)
  return max;
else if (value <= min)
  return min;
else 
  return clamp & round;
```

Reply:
@mattleibow sure, we can, but in this case we needn't have to call Clamp method here (max and min cases would be handled with if-else statements

Reply:
You are correct. Release the clamps and let the beast free!

Reply:
@mattleibow oh.. my code it totally wrong

For example: MinValue: 1, MaxValue : 7, Step: 2, Value: 3

Round(3 / 2) * 2 = 4 .... I need to rewrite it